virtcontainers/container.go

```diff
@@ -498,6 +498,15 @@ func (c *Container) shareFiles(m Mount, idx int, hostSharedDir, guestSharedDir s
 func (c *Container) mountSharedDirMounts(hostSharedDir, guestSharedDir string) (map[string]Mount, map[string]Mount, error) {
 	sharedDirMounts := make(map[string]Mount)
 	ignoredMounts := make(map[string]Mount)
+	var err error
```

Comment:
this won't work, because of `:=` creates a new variable so this variable will always be `nil`